index.test.js

```diff
@@ -97,4 +97,76 @@ tap.test('handles standard federal holidays', async tests => {
       );
     }
   );
+
+    tests.test(
+      'handles federal holidays within a range (Saturday and Sunday holidays shifted)',
+       async test => {
+        const holidays = federalHolidays.inRange({ start: new Date('2015-07-03'), end: new Date('2016-12-26') })
+        holidays.forEach(holiday => {
+          test.ok(
+            federalHolidays.isAHoliday(getDate(holiday.dateString)),
+           `${holiday.dateString} is a holiday (observed)`
```

Comment:
I like this, it's a clever approach to verify that all the dates are holidays.  Do you think it'd make sense to expand just this test out a little bit, though, and explicitly list the holidays you expect to see?  That way the test would catch the case where `isRange` leaves anything out.